src/CounterInterceptor/Counts.cs

```diff
@@ -0,0 +1,54 @@
+public record Counts
+{
+    public Counts()
+    {
+        clientExecutions = 0;
+        clientQueries = 0;
+        clientSignals = 0;
+        workflowExecutions = 0;
+        workflowSignals = 0;
+        workflowQueries = 0;
+        workflowChildExecutions = 0;
+        workflowActivityExecutions = 0;
+    }
+
```

Comment:
```suggestion
```

Not needed, this is the default value already of every field here.

Reply:
Fixed